TodoApp/test/TodoApp.Services.Tests/Comparers/ItemEqualityComparer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using NUnit.Framework.Constraints;
+using TodoApp.Contracts.Models;
+
+namespace TodoApp.Services.Tests.Comparers
+{
+    internal static class ItemEqualityComparerWrapper
```

Comment:
I did not notice before, sorry, however, the file should always have the same name as a class that is in it… Here, I would go with `ItemEqualityExtensions` (both the file and the class) - it is a custom to suffix classes with extensions this way and that a comparer is in it, that seems to be a more or less implementation detail…